.github/workflows/local-testnet.yml

```diff
@@ -34,7 +34,6 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
       - name: Install lighthouse
-        if: steps.cache-cargo.outputs.cache-hit != 'true'
```

Comment:
this change is needed?

Reply:
yeah, see discussion here: https://github.com/sigp/lighthouse/pull/3027#discussion_r810705873